### PR TITLE
shiftstack: add liveliness to mecha az0

### PR DESCRIPTION
**Pull request by EmilienM**
To improve our monitoring.

### CODE REVIEW

**Comment by EmilienM** [comment]
We'll need https://github.com/shiftstack/shiftstack-ci/pull/210

File: ci-operator/step-registry/shiftstack/liveliness/shiftstack-liveliness-commands.sh [hunk 1]
```diff
@@ -8,6 +8,12 @@ CLUSTER_NAME=$(<"${SHARED_DIR}/CLUSTER_NAME")
 OPENSTACK_EXTERNAL_NETWORK="${OPENSTACK_EXTERNAL_NETWORK:-$(<"${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK")}"
 # Recycling BASTION_FLAVOR as it's a small flavor we can re-use.
 TESTING_FLAVOR="${TESTING_FLAVOR:-$(<"${SHARED_DIR}/BASTION_FLAVOR")}"
+ZONES="${ZONES:-$(<"${SHARED_DIR}/ZONES")}"
+
+AZ_ARG=""
+if [ -n "${ZONES}" ]; then
+  AZ_ARG="-a ${ZONES}"
```
Comment (mandre) [inline ci-operator/step-registry/shiftstack/liveliness/shiftstack-liveliness-commands.sh:15]: What happens when we have multiple zones? Right now, it looks like it's going to explode in server.sh (based on what is currently in https://github.com/shiftstack/shiftstack-ci/pull/210/).
Should we spin a VM in each zone?

Comment (EmilienM) [reply ci-operator/step-registry/shiftstack/liveliness/shiftstack-liveliness-commands.sh]: for now let's just use the first zone in the list. Our CI clouds are not multi zones.

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-liveliness-mecha-az0

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-liveliness-mecha-az0

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-liveliness-mecha-az0

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-liveliness-mecha-az0

**Comment by EmilienM** [comment]
/pj-rehearse ack

It worked:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45134/rehearse-45134-periodic-ci-shiftstack-shiftstack-ci-main-liveliness-mecha-az0/1719734681047404544/artifacts/liveliness-mecha-az0/shiftstack-liveliness/build-log.txt

**Pull request merged**